std_msgs/Byte number
std_msgs/Byte object_position
std_msgs/Byte trial

---
